dashboard/test/models/pd/regional_partner_mini_contact_test.rb

```diff
@@ -0,0 +1,124 @@
+require 'test_helper'
+require 'testing/poste_assertions'
+
+class Pd::RegionalPartnerMiniContactTest < ActiveSupport::TestCase
+  include PosteAssertions
+
+  test 'can create a new regional partner mini contact' do
+    assert valid_form? build(
+      :pd_regional_partner_mini_contact_hash
+    )
+  end
+
+  test 'email is required' do
+    refute valid_form? build(
+      :pd_regional_partner_mini_contact_hash
+    ).merge("email" => "")
+  end
+
+  test 'zip is required' do
+    refute valid_form? build(
+      :pd_regional_partner_mini_contact_hash
+    ).merge("zip" => "")
+  end
+
+  test 'name is not required' do
+    refute_includes Pd::RegionalPartnerMiniContact.required_fields, :name
+  end
+
+  test 'notes is not required' do
+    refute_includes Pd::RegionalPartnerMiniContact.required_fields, :notes
+  end
+
+  private def valid_form?(form_data)
+    build(:pd_regional_partner_mini_contact, form_data: form_data.to_json).valid?
+  end
+
+  test 'Matches regional partner' do
+    state = 'OH'
+    zip = '45242'
```

Comment:
could you add a comment that this is the default zip code in the factory? otherwise it's not very clear why this test passes, since we aren't passing the zip as a parameter to the factory.